core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.ts

```diff
@@ -1147,6 +1197,10 @@ export class ConversationSkinComponent {
             new Date().getTime() - timeAtServerCall),
           1.0));
 
+        if (this.explorationPlayerStateService.isInDiagnosticTestPlayerMode()) {
```

Comment:
This should be an if/else with the above since they're mutually exclusive.

Why are we special-casing this, btw?

Reply:
Refactored to `if - else if - else` block.